memote/suite/cli/runner.py

```diff
@@ -51,6 +47,15 @@
     ResultManager, RepoResultManager, SQLResultManager, HistoryManager)
 from memote.utils import is_modified, stdout_notifications
 
+import requests
+from requests.exceptions import HTTPError
+import tempfile
+
+try:
+    from urllib.parse import quote_plus
```

Comment:
Since you use `six` throughout memote, could consider using https://pythonhosted.org/six/#module-six.moves here.